crud/operations.go

```diff
@@ -1,5 +1,7 @@
 package crud
 
+import "github.com/vmihailenco/msgpack/v5"
```

Comment:
I prefer that style in the repository (if this is not done by the autoformatter).
```suggestion
import (
    "github.com/vmihailenco/msgpack/v5"
)
```

Reply:
Updated.